<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Calculator</class>
 <widget class="QMainWindow" name="Calculator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>237</width>
    <height>334</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>237</width>
    <height>334</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Calculator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="4" column="2">
     <widget class="QPushButton" name="button3">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
background-color: #3B185F;
border: 1px solid grey;
padding: 5px;
color: #ffffff;
}

QPushButton:pressed {
background-color: rgb(122, 51, 243);
border: 1px solid grey;
padding: 5px;
color: #ffffff;
}



</string>
      </property>
      <property name="text">
       <string>3</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QPushButton" name="button2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
background-color: #3B185F;
border: 1px solid grey;
padding: 5px;
color: #ffffff;
}

QPushButton:pressed {
background-color: rgb(122, 51, 243);
border: 1px solid grey;
padding: 5px;
color: #ffffff;
}



</string>
      </property>
      <property name="text">
       <string>2</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QPushButton" name="button1">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
background-color: #3B185F;
border: 1px solid grey;
padding: 5px;
color: #ffffff;
}

QPushButton:pressed {
background-color: rgb(122, 51, 243);
border: 1px solid grey;
padding: 5px;
color: #ffffff;
}



</string>
      </property>
      <property name="text">
       <string>1</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="button7">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
background-color: #3B185F;
border: 1px solid grey;
padding: 5px;
color: #ffffff;
}

QPushButton:pressed {
background-color: rgb(122, 51, 243);
border: 1px solid grey;
padding: 5px;
color: #ffffff;
}



</string>
      </property>
      <property name="text">
       <string>7</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QPushButton" name="button5">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
background-color: #3B185F;
border: 1px solid grey;
padding: 5px;
color: #ffffff;
}

QPushButton:pressed {
background-color: rgb(122, 51, 243);
border: 1px solid grey;
padding: 5px;
color: #ffffff;
}



</string>
      </property>
      <property name="text">
       <string>5</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="buttonPercent">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
background-color: #3B185F;
border: 1px solid grey;
padding: 5px;
color: #ffffff;
}

QPushButton:pressed {
background-color: rgb(122, 51, 243);
border: 1px solid grey;
padding: 5px;
color: #ffffff;
}



</string>
      </property>
      <property name="text">
       <string>%</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QPushButton" name="button9">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
background-color: #3B185F;
border: 1px solid grey;
padding: 5px;
color: #ffffff;
}

QPushButton:pressed {
background-color: rgb(122, 51, 243);
border: 1px solid grey;
padding: 5px;
color: #ffffff;
}



</string>
      </property>
      <property name="text">
       <string>9</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="button4">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
background-color: #3B185F;
border: 1px solid grey;
padding: 5px;
color: #ffffff;
}

QPushButton:pressed {
background-color: rgb(122, 51, 243);
border: 1px solid grey;
padding: 5px;
color: #ffffff;
}



</string>
      </property>
      <property name="text">
       <string>4</string>
      </property>
     </widget>
    </item>
    <item row="4" column="3">
     <widget class="QPushButton" name="buttonDivide">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
background-color: #3B185F;
border: 1px solid grey;
padding: 5px;
color: #ffffff;
}

QPushButton:pressed {
background-color: rgb(122, 51, 243);
border: 1px solid grey;
padding: 5px;
color: #ffffff;
}



</string>
      </property>
      <property name="text">
       <string>รท</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QPushButton" name="buttonSubtract">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
background-color: #3B185F;
border: 1px solid grey;
padding: 5px;
color: #ffffff;
}

QPushButton:pressed {
background-color: rgb(122, 51, 243);
border: 1px solid grey;
padding: 5px;
color: #ffffff;
}



</string>
      </property>
      <property name="text">
       <string>-</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="button8">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
background-color: #3B185F;
border: 1px solid grey;
padding: 5px;
color: #ffffff;
}

QPushButton:pressed {
background-color: rgb(122, 51, 243);
border: 1px solid grey;
padding: 5px;
color: #ffffff;
}



</string>
      </property>
      <property name="text">
       <string>8</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="buttonPlusMinus">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
background-color: #3B185F;
border: 1px solid grey;
padding: 5px;
color: #ffffff;
}

QPushButton:pressed {
background-color: rgb(122, 51, 243);
border: 1px solid grey;
padding: 5px;
color: #ffffff;
}



</string>
      </property>
      <property name="text">
       <string>+/-</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="buttonC">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
background-color: #3B185F;
border: 1px solid grey;
padding: 5px;
color: #ffffff;
}

QPushButton:pressed {
background-color: rgb(122, 51, 243);
border: 1px solid grey;
padding: 5px;
color: #ffffff;
}



</string>
      </property>
      <property name="text">
       <string>C</string>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QPushButton" name="buttonMultiply">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
background-color: #3B185F;
border: 1px solid grey;
padding: 5px;
color: #ffffff;
}

QPushButton:pressed {
background-color: rgb(122, 51, 243);
border: 1px solid grey;
padding: 5px;
color: #ffffff;
}



</string>
      </property>
      <property name="text">
       <string>x</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QPushButton" name="button6">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
background-color: #3B185F;
border: 1px solid grey;
padding: 5px;
color: #ffffff;
}

QPushButton:pressed {
background-color: rgb(122, 51, 243);
border: 1px solid grey;
padding: 5px;
color: #ffffff;
}



</string>
      </property>
      <property name="text">
       <string>6</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QPushButton" name="buttonAdd">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
background-color: #3B185F;
border: 1px solid grey;
padding: 5px;
color: #ffffff;
}

QPushButton:pressed {
background-color: rgb(122, 51, 243);
border: 1px solid grey;
padding: 5px;
color: #ffffff;
}



</string>
      </property>
      <property name="text">
       <string>+</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0" colspan="2">
     <widget class="QPushButton" name="button0">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
background-color: #3B185F;
border: 1px solid grey;
padding: 5px;
color: #ffffff;
}

QPushButton:pressed {
background-color: rgb(122, 51, 243);
border: 1px solid grey;
padding: 5px;
color: #ffffff;
}



</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="5" column="2">
     <widget class="QPushButton" name="buttonDecimal">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
background-color: #3B185F;
border: 1px solid grey;
padding: 5px;
color: #ffffff;
}

QPushButton:pressed {
background-color: rgb(122, 51, 243);
border: 1px solid grey;
padding: 5px;
color: #ffffff;
}



</string>
      </property>
      <property name="text">
       <string>.</string>
      </property>
     </widget>
    </item>
    <item row="5" column="3">
     <widget class="QPushButton" name="buttonEquals">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
background-color: #3B185F;
border: 1px solid grey;
padding: 5px;
color: #ffffff;
}

QPushButton:pressed {
background-color: rgb(122, 51, 243);
border: 1px solid grey;
padding: 5px;
color: #ffffff;
}



</string>
      </property>
      <property name="text">
       <string>=</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="4">
     <widget class="QLineEdit" name="display">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>36</pointsize>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
background-color: rgb(104, 104, 104);
border: 1px solid gray;
color: #ffffff;
padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>237</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
